tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects.ts

```diff
@@ -62,6 +131,77 @@ export class DebuggerEffects {
     )
   );
 
+  /** @export */
+  readonly loadNumExecutions$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(activeRunIdChanged),
+      withLatestFrom(this.store.select(getNumExecutionsLoaded)),
+      filter(([, loaded]) => loaded.state !== DataLoadState.LOADING),
+      tap(() => this.store.dispatch(numExecutionsRequested())),
+      mergeMap((props) => {
+        const runId = props[0].activeRunId as string; // TODO(cais): Guard against null?
```

Comment:
`// TODO(cais): Guard against null?` Can we do this in the `filter` above? Also, what's the condition for this?